python/tchannel/io.py

```diff
@@ -19,8 +19,169 @@
 # THE SOFTWARE.
 
 from __future__ import absolute_import
+import os
+from .exceptions import StreamingException
+from .messages.common import StreamState
+
 
 try:
     from cStringIO import StringIO as BytesIO
 except ImportError:  # pragma: no cover
     from io import BytesIO  # noqa
+
+import tornado
+import tornado.gen
+import tornado.ioloop
+from tornado.iostream import PipeIOStream, StreamClosedError
+
+try:
+    from tornado.locks import Condition
+except ImportError:  # pragma: no cover
+    from toro import Condition
+
+from .messages import common
+
+
+class BaseArgStream(object):
+
+    def __init__(self):
+        """Base Arg Stream Class
```

Comment:
These words don't mean anything and don't provide any information that isn't already available from the class name. I would get rid of the whole __init__ docstring.